app/src/main/java/com/keylesspalace/tusky/components/report/fragments/ReportDoneFragment.kt

```diff
@@ -49,11 +47,11 @@ class ReportDoneFragment : Fragment(R.layout.fragment_report_done) {
             viewModel.muteState.collect {
                 if (it == null) return@collect
                 if (it !is Loading) {
-                    binding.buttonMute.show()
-                    binding.progressMute.show()
+                    binding.buttonMute.visibility = View.VISIBLE
+                    binding.progressMute.visibility = View.GONE
```

Comment:
Is it intentional to change the behavior here? Was it wrong?

Reply:
Yes intentional because of two reasons:
- The `CircularProgressIndicator` has methods `show()` / `hide()` that don't do exactly what is expected, causing #4692 
- Setting the buttons to invisible instead of gone doesn't reflow the layout which is nicer